nss-day-cohort-45/nutyshell-hitting-your-siblings
# Jic nutshell component

# Description
User auth and then render using nutshell

# Testing Instructions
Fetch branch and ensure it renders the login and register
After login/register should clear boxes and render events, news, tasks
# Checklist:
- [X ] My code follows the style guidelines of this project
- [ X] I have performed a self-review of my own code
- [ X] My changes generate no new warnings or errors
- [ X] I have added test instructions that prove my fix is effective or that my feature works
